common-rest/src/main/java/org/uniprot/api/common/repository/search/QueryBoostsFileReader.java

```diff
@@ -0,0 +1,97 @@
+package org.uniprot.api.common.repository.search;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.stream.Stream;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Created 04/09/19
+ *
+ * @author Edd
+ */
+@Slf4j
+public class QueryBoostsFileReader {
```

Comment:
Do you think we could use spring configuration logic, like we do with facet? I think each type would be an attribute in the @Configuration mapped class.